lib/msf/util/payload_cached_size.rb

```diff
@@ -14,6 +14,27 @@ module Util
 
 class PayloadCachedSize
 
+  @opts = {
```

Comment:
I don't think it has sense as instance variable. Indeed, I don't think we can't create `PayloadCachedSize` even :). I'm going to switch to a constant before landing if you don't mind. Looks definitely like a constant. Feel free to fix me if I'm wrong, just reviewing changes :)